src/mongo/db/mongod_options.cpp

```diff
@@ -62,6 +62,11 @@ using std::string;
 MongodGlobalParams mongodGlobalParams;
 
 extern DiagLog _diaglog;
+extern int cursorTimeoutMillis;
```

Comment:
This won't link as these variables are declared `extern` and need to be defined elsewhere.
Maybe better to create a separate file like `percona_options.h/cpp` and declare our own `PerconaGlobalParams` struct there?